addons/website_mail/models/mail_message.py

```diff
@@ -24,6 +24,7 @@ def default_get(self, fields_list):
         return defaults
 
     description = fields.Char(compute="_compute_description", help='Message description: either the subject, or the beginning of the body')
+    # TODO: too violent to use mixin here?
```

Comment:
mail.message is probably the model which has the higher number of record, so for perf issue I guess it is better not to use the mixin?

Reply:
Yeah that's what I thought